parachains/integration-tests/e2e/collectives/collectives-polkadot/3_fellowship/1_whitelist_call.yml

```diff
@@ -0,0 +1,146 @@
+      wsPort: 9710
+      paraId: &cp_id 1001
+  variables:
+    fellows_proposal_index: &fellows_proposal_index 0
+    chains:
+      accounts:
+        alice_signer: &alice_signer //Alice
+  decodedCalls:
+    remark_rc:
+      chain: *relay_chain
+      encode: false
+      pallet: system
+      call: remark
+      args: [
+        "0x10"
+      ]
+    whitelist_remark_rc:
+      chain: *relay_chain
+      encode: true
+      pallet: whitelist
+      call: whitelistCall
+      args: [
+        $remark_rc.hash
+      ]
+    send_whitelist_remark_cp:
+      chain: *collectives_parachain
+      encode: false
+      pallet: polkadotXcm
+      call: send
+      args: [
+        { v3: { parents: 1, interior: { here: true }}}, # destination
+        {
+          v3: [ #message
+            {
+              UnpaidExecution: {
+                weightLimit: Unlimited
+              }
+            },
+            {
+              Transact: {
+                originKind: Xcm,
+                requireWeightAtMost: {
+                  refTime: 500000000, # 500_000_000
+                  proofSize: 20000, # 20_000
+                },
+                call: $whitelist_remark_rc
+              }
+            }
+          ]
+        }
+      ]
+
+tests:
+  - name: Fellowship
+    describes:
+      - name: The Fellowship white list the call
+        its:
+          - name: Note preimage from the whitelist call on the Relay Chain
+            actions:
+              - extrinsics:
+                - chain: *relay_chain
+                  signer: *alice_signer
+                  pallet: preimage
+                  call: notePreimage
+                  args: [
+                    $remark_rc
+                  ]
+                  events:
+                    - name: preimage.Noted
+                      result: { hash_: $remark_rc.hash }
+          - name: Note preimage from the xcm send call to white list the call above
+            actions:
+              - extrinsics:
+                - chain: *collectives_parachain
+                  signer: *alice_signer
+                  pallet: preimage
+                  call: notePreimage
+                  args: [
+                    $send_whitelist_remark_cp,
+                  ]
+                  events:
+                    - name: preimage.Noted
+                      result: { hash_: $send_whitelist_remark_cp.hash }
+          - name: Submit a proposal to while list the call
+            actions:
+              - extrinsics:
+                - chain: *collectives_parachain
+                  signer: *alice_signer
+                  pallet: fellowshipReferenda
+                  call: submit
+                  args: [
+                    {
+                      "FellowshipOrigins": "Fellows",
+                    },
+                    {
+                      "Lookup": { # TODO obtain from the call https://github.com/paritytech/parachains-integration-tests/issues/103
```

Comment:
nice, looks like this can be simplified now that nacho's improved things.

Reply:
That TODO can be removed then, right?

Reply:
yes, this already addressed

Reply:
removed